frontend/src/metabase/components/ExplicitSize.jsx

```diff
@@ -10,7 +10,7 @@ import { isCypressActive } from "metabase/env";
 
 // After adding throttling for resize re-renders, our Cypress tests became flaky
 // due to queried DOM elements are getting detached after re-renders
-const throttleDuration = isCypressActive ? 0 : 500;
+const throttleDuration = isCypressActive ? 0 : 300;
```

Comment:
@WiNloSt Wow - what a difference, and great explanation. From the video, it looks like you're also fixing #16704, but I haven't taken it for a spin myself yet.

Reply:
@flamber I also noticed that some transitions took longer than expected. Seems like the flickers are replaced by a longer transition. e.g. instead of the resize transition happening 2 times, it only occurs once but seems to last the same amount of time as the transition flickers 2 times. This kind of gives me an impression that the app is slower IMO, but it won't flicker, and in the end, took the same amount of time until the UI is interactive.

But at least this change will definitely remove other bugs around `ExplicitSize` or `DebouncedFrame` or at least make other bugs easier to discover by making the UI state more consistent.